category_theory/examples/topological_spaces.lean

```diff
@@ -27,6 +29,57 @@ instance : concrete_category @continuous := ⟨@continuous_id, @continuous.comp
 
 -- local attribute [class] continuous
 -- instance {R S : Top} (f : R ⟶ S) : continuous (f : R → S) := f.2
+
+section
+open category_theory.limits
+
+variables {J : Type u} [small_category J]
+
+def limit (F : J ⥤ Top.{u}) : cone F :=
+{ X := ⟨limit (F ⋙ forget), ⨆ j, (F.obj j).str.induced (limit.π (F ⋙ forget) j)⟩,
+  π :=
+  { app := λ j, ⟨limit.π (F ⋙ forget) j, continuous_iff_induced_le.mpr (lattice.le_supr _ j)⟩,
+    naturality' := λ j j' f, subtype.eq ((limit.cone (F ⋙ forget)).π.naturality f) } }
+
+def limit_is_limit (F : J ⥤ Top.{u}) : is_limit (limit F) :=
+{ lift := λ s,
+    ⟨limit.lift (F ⋙ forget) (forget.map_cone s),
+     continuous_iff_le_coinduced.mpr $ lattice.supr_le $ λ j,
+       induced_le_iff_le_coinduced.mpr $ continuous_iff_le_coinduced.mp (s.π.app j).property⟩,
+  fac' := λ s j, subtype.eq rfl,
```

Comment:
Actually, maybe it's worth factoring out these last three lines and proving some lemma about when a cone is a limit cone if its image under a faithful functor is one.